examples/dynamoathenamigration/migration.go

```diff
@@ -361,9 +362,15 @@ func (t *task) getDataObjectsInfo(ctx context.Context, manifestPath string) ([]d
 }
 
 func (t *task) getEventsFromDataFiles(ctx context.Context, exportInfo *exportInfo, eventsC chan<- apievents.AuditEvent) error {
+	// TODO(Joerger): Rather than loading the checkpoint and prompting the user after completing the export step,
+	// this should be done at the beginning of program execution.
 	checkpoint, err := t.loadEmitterCheckpoint(ctx, exportInfo.ExportARN)
 	if err != nil {
-		return trace.Wrap(err)
+		t.Logger.InfoContext(ctx, "Failed to load checkpoint file from previous migration attempt", "file", t.CheckpointPath, "error", err)
+		_, err := prompt.Confirmation(ctx, os.Stdout, prompt.Stdin(), fmt.Sprintf("It seems that a previous migration %s stopped with error and there was an issue resuming it. Would you like to start a new migration?", exportInfo.ExportARN))
```

Comment:
Should we mention error message in this prompt? So that the user have more informed knowledge on the previous failure before starting a new migration.

Reply:
I didn't want to overcrowd the prompt, so I moved the error message into it's own INFO log, which is the default log level of this tool.